egs/vcc2020/vc1_task1/run.sh

```diff
@@ -0,0 +1,459 @@
+#!/bin/bash
+
+# Copyright 2020 Nagoya University (Wen-Chin Huang)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# general configuration
+backend=pytorch
+stage=-1
+stop_stage=100
+ngpu=1       # number of gpus ("0" uses cpu, otherwise use gpu)
+nj=10        # numebr of parallel jobs
+dumpdir=dump # directory to dump full features
+verbose=0    # verbose option (if set > 0, get more log)
+N=0          # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+seed=1       # random seed number
+resume=""    # the snapshot path to resume (if set empty, no effect)
+
+# feature extraction related
+fs=24000      # sampling frequency
+fmax=7600     # maximum frequency
+fmin=80       # minimum frequency
+n_mels=80     # number of mel basis
+n_fft=1024    # number of fft points
+n_shift=256   # number of shift points
+win_length="" # window length
+
+# silence part trimming related
+trim_threshold=25 # (in decibels)
+trim_win_length=1024
+trim_shift_length=256
+trim_min_silence=0.01
+
+trans_type=char  # char or phn
+
+# config files
+train_config=conf/train_pytorch_transformer+spkemb.yaml
+decode_config=conf/decode.yaml
+
+# decoding related
+model=model.loss.best
+voc=                            # GL or PWG
+voc_expdir=downloads/pwg_task1  # If use provided pretrained models, set to desired dir, ex. `downloads/pwg_task1`
+                                # If use manually trained models, set to `../voc1/exp/<expdir>`
+voc_checkpoint=                 # If not specified, automatically set to the latest checkpoint 
+griffin_lim_iters=64            # the number of iterations of Griffin-Lim
+
+# pretrained model related
+pretrained_model_dir=downloads  # If use provided pretrained models, set to desired dir, ex. `downloads`
+                                # If use manually trained models, set to `../libritts`
+pretrained_model_name=          # Recommended choices: tts1
+
+# dataset configuration
+db_root=downloads/official_v1.0_training
+eval_db_root=downloads/official_v1.0_training    # Same as `db_root` in training
+list_dir=local/lists
+spk=TEF1 
+
+# vc configuration
+srcspk=
+trgspk=
```

Comment:
And how about adding the check whether to correctly set these variables which needed to set manually in the beginning of the conversion stage?

Reply:
> And how about adding the check whether to correctly set these variables which needed to set manually in the beginning of the conversion stage?

I can add the comments. As for argument checking, I found that in some ESPnet recipes the check does not always exist, and since we set bash to debug mode, any undefined argument (variable?) will cause the program to instantly exit. So...